src/spentbook.rs

```diff
@@ -214,7 +221,7 @@ impl SpentBookNodeMock {
             })
         } else {
             // fixme: return an error.  can wait until we refactor into a Mock feature flag.
```

Comment:
rm this fixme?

Reply:
will remove it in coming days while doing pass of fixme/todo/unwraps.  polishing, etc.